cmake/Corrosion.cmake

```diff
@@ -1092,6 +1092,28 @@ function(corrosion_link_libraries target_name)
     endforeach()
 endfunction()
 
+#[=======================================================================[.md:
+ANCHOR: corrosion-install
+** EXPERIMENTAL **: This function is currently still considered experimental
+  and is not officially released yet. Feedback and Suggestions are welcome.
+
+```cmake
+corrosion_install(TARGETS <target1> ... <targetN>
+                  [[ARCHIVE|LIBRARY|RUNTIME]
+                   [DESTINATION <dir>]
+                   [PERMISSIONS <permissions...>]
+                   [CONFIGURATIONS [Debug|Release|<other-configuration>]]
+                  ] [...])
+```
+* **TARGETS**: Target or targets to install.
+* **EXPORT**: NOT IMPLEMENTED.
+
+TODO: The `ARCHIVE`/`LIBRARY`/... nonsense works like `<artifact-kind> <artifact-option>...` from [`install(TARGETS)`].
```

Comment:
Could you reword this sentence? Linking to the CMake docs is fine, but the wording should be fixed before merging.

Reply:
Removed TODO and reworded other parameters.

I wasn't sure how final the documentation was for unreleased functions, but I understand keeping it relatively clean.

I intend on creating a PR for installing headers soon-ish as well.